lib/move/__tests__/move.test.js

```diff
@@ -331,6 +331,25 @@ describe('+ move()', () => {
     })
   })
 
+  describe('> passing undefined explicitly for opts in promise mode', () => {
+    it('should not produce a TypeError', done => {
+      const SRC_DIR = path.join(TEST_DIR, 'test')
+      const DEST_DIR = path.join(TEST_DIR, 'foobar')
+
+      fs.mkdirSync(SRC_DIR)
+
+      fse.move(SRC_DIR, DEST_DIR, undefined)
+        .then(() => done())
+        .catch(error => {
+          if (error instanceof TypeError) {
+            done()
+          } else {
+            done(error)
+          }
+        })
```

Comment:
Currently, the test would pass if a `TypeError` was thrown; fix:
```suggestion
        .catch(done)
```